Dockerfile

```diff
@@ -8,11 +8,10 @@ WORKDIR /opt/certbot
 COPY CHANGELOG.md README.rst setup.py src/
 
 # Generate constraints file to pin dependency versions
-COPY letsencrypt-auto-source/pieces/dependency-requirements.txt hashed_requirements.txt
+COPY letsencrypt-auto-source/pieces/dependency-requirements.txt .
 COPY tools /opt/certbot/tools
-RUN /opt/certbot/tools/docker_constraints.py \
-    hashed_requirements.txt unhashed_requirements.txt \
-    tools/dev_constraints.txt docker_constraints.txt
+RUN sh -c 'cat dependency-requirements.txt | /opt/certbot/tools/strip_hashes.py > unhashed_requirements.txt'
+RUN sh -c 'cat tools/dev_constraints.txt unhashed_requirements.txt | /opt/certbot/tools/merge_requirements.py > docker_constraints.txt'
```

Comment:
I don't think this works with `merge_requirements.py`. It currently takes a list of filenames as CLI arguments.